.github/workflows/test.yaml

```diff
@@ -183,14 +183,6 @@ jobs:
           echo
           df -h
 
-      - name: Smoke test - two_party_countersigning
```

Comment:
Would commenting out be preferred rather than removing? I thought given that we likely won't bother about the scenario for quite a while removing is cleaner.